packages/button-theme/src/Notification.tsx

```diff
@@ -28,156 +30,53 @@ export const styles = {
   },
   size: {
     small: {
+      ...styles.size.small,
       container: `
         padding: 0.8rem 0.9rem;
       `,
-      header: `
-        font-size: 1rem;
-      `,
+      // header: `
```

Comment:
are these left in but commented out as examples of styles that can be overriden?

Reply:
nope, just forgot to push